smoke-tests/src/test/java/com/example/javaagent/smoketest/UserAgentHeaderTest.java

```diff
@@ -188,7 +188,7 @@ private GenericContainer<?> runTestApp(Consumer<GenericContainer<?>> customizer)
             .withEnv("OTEL_BSP_SCHEDULE_DELAY", "10")
             .withEnv("OTEL_BLRP_MAX_EXPORT_BATCH", "1")
             .withEnv("OTEL_BLRP_SCHEDULE_DELAY", "10")
-            .withEnv("OTEL_METRIC_EXPORT_INTERVAL", "10")
```

Comment:
can you add a comment with a link to the issue so we can later refer to this when trying to properly fix the underlying issue ?

Reply:
Done in https://github.com/elastic/elastic-otel-java/pull/643/commits/2914b9d4ea356a202601c6e5601c3d2ef5abef97